### PR TITLE
Fix #2562: Fixed parameters in the comments doesn't match with the actual parameters.

### LINKED ISSUE #2562: Parameters in the comments doesn't match with the actual parameters
The parameters mentioned in the comments don't match with the actual parameters in `StoryProgressTestHelper`

For example here the comment(timestampOlderThanOneWeek) doesn't match with the actual parameter(timestampOlderThanAWeek)
https://github.com/oppia/oppia-android/blob/9bc61d89c950dde5cf0b5be0b6232dc16e2c6357/domain/src/main/java/org/oppia/android/domain/topic/StoryProgressTestHelper.kt#L23-L29

### PR DESCRIPTION
**Pull request by OBITORASU**
## Explanation

"Fixes #2562:  I have updated all the comments in ```oppia-android/domain/src/main/java/org/oppia/android/domain/topic/StoryProgressTestHelper.kt``` to match the parameters actually being passed.
 

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Review by FareesHussain — APPROVED** [review]
LGTM, thanks

File: domain/src/main/java/org/oppia/android/domain/topic/StoryProgressTestHelper.kt [hunk 1]
```diff
@@ -24,7 +24,7 @@ class StoryProgressTestHelper @Inject constructor(
    * Creates a partial story progress for a particular profile.
    *
    * @param profileId the profile we are setting partial progress of the fraction story for
-   * @param timestampOlderThanOneWeek if the timestamp for this topic progress is more than one week ago
```
Comment (BenHenning) [inline domain/src/main/java/org/oppia/android/domain/topic/StoryProgressTestHelper.kt:27]: Actually, can we go the other way & update the parameter to match this name, instead? I think 'one week' is clearer than 'a week'.

**Review by BenHenning — CHANGES_REQUESTED** [review]
Thanks @OBITORASU! This looks correct, but I'm actually thinking we can make the overall code a bit clearer. Left one comment.

File: domain/src/main/java/org/oppia/android/domain/topic/StoryProgressTestHelper.kt (hunk 1 above)
Comment (OBITORASU) [reply domain/src/main/java/org/oppia/android/domain/topic/StoryProgressTestHelper.kt:27]: Sure thing, first thing I do when I hop onto my workstation.

**Review by FareesHussain — CHANGES_REQUESTED** [review]
@OBITORASU please change `timestampOlderThanOneweek` to `timestampOlderThanOneWeek`

Also resolve the errors in the following files 
 - `StoryProgressTestHelperTest`
 - `HomeActivityTest`
 - `RecentlyPlayedFragmentTest`
 - `OngoingTopicListActivityTest`
 - `ProfileProgressFragmentTest`
 - `StoryFragmentTest`
 - `NavigationDrawerActivityTest`
 - `TopicLessonsFragmentTest`
 - `RecentlyPlayedSpanTest`

**Comment by OBITORASU** [comment]
@FareesHussain sorry about this noob question but to fix the issues in the files you referenced, I have to update all the ```timestampOlderThanAWeek``` to ```timestampOlderThanOneWeek``` right?

**Comment by FareesHussain** [comment]
Yes

**Comment by OBITORASU** [comment]
Thank you I will start fixing them right away.

**Review by FareesHussain — CHANGES_REQUESTED** [review]
I've missed this in the last review.
Please resolve the errors in `CompletedStoryListActivityTest` too.

File: domain/src/main/java/org/oppia/android/domain/topic/StoryProgressTestHelper.kt (hunk 1 above)
Comment (anandwana001) [reply domain/src/main/java/org/oppia/android/domain/topic/StoryProgressTestHelper.kt:27]: The PR reviewer can resolve the conversation after confirming the suggested changes are done or if anything left to discuss. The conversation should not be resolved by PR authors.

Comment (OBITORASU) [reply domain/src/main/java/org/oppia/android/domain/topic/StoryProgressTestHelper.kt:27]: Resolved!

Comment (OBITORASU) [reply domain/src/main/java/org/oppia/android/domain/topic/StoryProgressTestHelper.kt:27]: Sorry my bad.

**Review by FareesHussain — APPROVED** [review]
LGTM, thanks

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Review by BenHenning — APPROVED** [review]
Thanks @OBITORASU! LGTM!

File: domain/src/main/java/org/oppia/android/domain/topic/StoryProgressTestHelper.kt (hunk 1 above)
Comment (BenHenning) [reply domain/src/main/java/org/oppia/android/domain/topic/StoryProgressTestHelper.kt:27]: No worries. :) Thanks for fixing this @OBITORASU!

**Review by anandwana001 — APPROVED** [review]
Approved as CODEOWNER.

Need to resolve conflict.

**Comment by OBITORASU** [comment]
Done!

**Pull request merged**